src/index.html

```diff
@@ -17,11 +17,438 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./styles/index.scss"
     />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#">
+          <img
+            src="../src/images/logo.png"
+            alt="logo"
+            class="header__logo-img"
+          />
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list-item">
+            <a
+              href="#apple"
+              class="nav__list-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__list-item">
```

Comment:
Add the blank line between the links
```suggestion
          </li>
          
          <li class="nav__list-item">
```